utils/node_local_test.py

```diff
@@ -440,10 +440,12 @@ def dfuse_mount_name(self):
 class DaosServer():
     """Manage a DAOS server instance"""
 
-    def __init__(self, conf, test_class=None, valgrind=False):
+    def __init__(self, conf, test_class=None, valgrind=False, wf=None, fe=None):
         self.running = False
         self._file = __file__.lstrip('./')
         self._sp = None
+        self.wf = wf
+        self.fe = fe
```

Comment:
If you have to push again, could you please add either a docstring or inline comment for these params? Specifically, `wf` and `fe` aren't immediately intuitive (to me, at least). I think `we=WarningsFactory` and `fe=fatal_errors`?

Reply:
For the pylint warnings I was planning to address them (in fact it's the warnings that made me aware that I could make the improvements in this PR).  I've not had time to do this though, and am wary of conflicts with so many changes in flight but I think you're right and we should just wholesale silence the noise to improve quality of reporting.

Yes to these variable names.  I'll make the suggested changes if I need to repush.

Reply:
Makes sense, and thanks!